mediator/esmFldsExchange_coastal_mod.F90

```diff
@@ -458,8 +557,161 @@ subroutine esmFldsExchange_coastal_init(gcomp, phase, rc)
       end do
       deallocate(S_flds)
     end if
+    
+    
+    !======================================================================
+    ! BEGIN FIELDS TO ICE (compice)
+    !======================================================================
+    ! ---------------------------------------------------------------------                      
+    ! To ice from atm
+    ! ---------------------------------------------------------------------                      
+    ! ----------------------------------------  
+    ! | Atmospheric Fields:                  |
+    ! ----------------------------------------
+    !   - zonal wind height 10m     [m/s]
+    !   - merid wind height 10m     [m/s]
+    !   - lowest atmospheric height [m] (defualt in CICE is 10m see model comp)
+    !   - Temperature at 10m        [k]
+    !   - Specific Humidity         [kg/kg]
+    !   - Pressure at 10m           [Pa]
+    !   - Rainfall rate             [kg/m/m/s]
+    !   - Snowfall rate             [kg/m/m/s]
+    !   - Net shortwave rad         [W/m/m] Other bands are derived from this
+    !   - Net longwave  rad         [W/m/m] 
+    ! ---------------------------------------
+    if (coastal_attr%atm_present .and. coastal_attr%ice_present) then                            
+      allocate(S_flds(14))                                                                       
+      S_flds = (/'Sa_u10m',    & ! inst_zonal_wind_height10m
```

Comment:
@SmithJos13 Again, all strings needs to be in same length. Please add space to some variables to fix it.